src/strings/trie/trie_static.cpp

```diff
@@ -32,6 +35,7 @@ int addEdge(int id, char c) {
     int& nxt = trie[id][c];
     if (nxt == -1) {
         nxt = ++nodesCount;
```

Comment:
This is already set since `nxt` is just a reference to `trie[id][c]`.

Reply:
sorry, this was a misunderstanding on my part. Removing it from the changes